javatests/arcs/core/storage/RamDiskBackingStoreIntegrationTest.kt

```diff
@@ -68,7 +68,7 @@ class RamDiskBackingStoreIntegrationTest {
         val count2 = CrdtCount()
         count2.applyOperation(MultiIncrement("them", version = 0 to 10, delta = 15))
 
-        val id = store.on(MultiplexedProxyCallback { _, _ -> true })
+        val id = store.on(MultiplexedProxyCallback { _, _ -> })
```

Comment:
No need for the args and arrow, you can just use empty braces `{ }`

Reply:
I had tried that; passing `{}` to `MultiplexedProxyCallback` here results in a compiler error for me. It seems that parameter omission is only supported when there is one argument.

I can, however, simply omit this `store.on` call completely.